Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/UserSignupViewController.m

```diff
@@ -76,11 +65,30 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         //nonprofit property is still nil
         NonprofitSignupViewController *nonprofitSignupVC = [segue destinationViewController];
         nonprofitSignupVC.user = self.user;
+        nonprofitSignupVC.nonprofit = self.user.nonprofit;
+        
     }
     
 
 }
 
+- (void)createUserToSave {
+    if (!self.user) {
+        self.user = [User user];
+    }
+    self.user.firstName = (self.firstNameTextField.text.length == 0) ? @"" : self.firstNameTextField.text;
```

Comment:
You can use the Elvis operator here and do `self.user.firstName = self.firstNameTextField.text ?: @"";`